docs/concepts.md

```diff
@@ -1,67 +1,69 @@
 # Concepts
 
-- [Runbook](#runbook)
-- [Node and Environment](#node-and-environment)
-- [Tools and Scripts](#tools-and-scripts)
-- [Platform](#platform)
-- [Feature](#feature)
-- [Requirement and Capability](#requirement-and-capability)
-- [Test suite and test case](#test-suite-and-test-case)
+- [Concepts](#concepts)
```

Comment:
You can add oneline in settings.json of vscode, it can automatically remove first level, and following other rules.
`    "markdown.extension.toc.levels": "2..6",`